src/styles/app.styl

```diff
@@ -80,6 +80,11 @@ body
 .lists-wrapper
     display flex
     flex-direction column
+    padding-left 1rem
+    padding-right 1rem
+    .MuiDivider-root
```

Comment:
Does it works on production build? Shouldn't we use a className here ? Or makeStyle? cc @JF-Cozy

Reply:
🤔 Je pense qu'en prod ces classes deviennent des `jss-xxx` ou quelque chose comme ça.
💡 L'idée serait peut-être plutôt de rajouter des `u-mv-1` sur le divider en question ?

Reply:
Ca ne fonctionnait pas mais la différence n'était pas flagrante. Va pour u-mv-1, merci @JF-Cozy je ne suis pas très au fait de ces classes utilitaires

Reply:
N'hésite pas à regarder si ton besoin est couvert par une classe utilitaire. Tu as toute la liste ici https://docs.cozy.io/cozy-ui/styleguide/section-utilities.html